### DISCUSSION TITLE
Software deploying

### DISCUSSION THREAD

**Discussion by marxxtone** [glpi-project/glpi-agent#504] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

I need help with deploying software over P2P using Agent 1.5 on LAN or GLPI server to target Windows 10 PC with Agent 1.5

I have GLPI 1.0.9 Linux server, GLPI inventory and deplyoment agents modul is activated, P2P is enabled by default on target PCs with Agent 1.5

I have a .msi file ready to install on endpoint in entity.

Package deployment:

Files: "googlechromestandaloneenterprise64.msi"
Actions: "msiexec /i googlechromestandaloneenterprise64.msi /quiet"

Tasks:

Task management: active, click on force start

Job configuration: Target and Actor are defined, Method Package deploy

Job executions: Prepared 1, Running 1, Cancelled 0, Not done yet 1, Successful 0, In error 0

I use SSL fingerprint for all Agents 1.5, Inventoring works well

LOG: [Wed Sep 27 07:01:15 2023][info] New inventory from DOMAINNAME-2023-08-22-14-42-30 for server0
[Wed Sep 27 07:02:03 2023][info] running task Deploy
[Wed Sep 27 07:02:03 2023][info] No Deploy job enabled or Deploy support disabled server side.
[Wed Sep 27 07:02:05 2023][info] running task Collect
[Wed Sep 27 07:02:05 2023][info] No Collect job enabled or Collect support disabled server side.
[Wed Sep 27 07:02:10 2023][info] target server0: next run: Thu Sep 28 06:57:23 2023 - https://DOMAINNAME/marketplace/glpiinventory/
[Wed Sep 27 07:02:10 2023][info] GLPI Agent memory usage: WSS=3448832 PFU=138407936

After clicking on Task Inventory GLPI Agent 1.5 on endpoint PC, starts only task Maintenance and waiting. No task Deploy. Is that ok?

I don't see any installation attempt on the end station and I don't see the application itself

Thanx for help

Mark

**Comment by g-bougard** [comment]
Hi @marxxtone 
if the agent doesn't run the deploy task, it means glpi-inventory plugin doesn't tell the agent it is concerned by the task.
There maybe few issue that can prevent this to happen.
The first you have to do is to restart the agent after you enabled debug2 level in registry. Then force again the task in GLPI and force the agent to run. Then check the log for any suspicious line.
Anyway, you still can check few things in GLPI:
 - check if you have an url set as "Agent base url" in the "Assets" tab on your entities. If it is wrong, the agent won't be able to find the job that has been prepared for it.
 - check the task configuration: the agent you're testing should be set as target or has to be included in the computer group you're targetting

**Comment by marxxtone** [comment]
OK, I've added the address to the assets, it was missing.
The target is selected as a separate PC in the LDAP directory.
Tomorrow I'll turn on debug2, look at the logs and write. 
Thank you very much!

![image](https://github.com/glpi-project/glpi-agent/assets/145987827/6eb5edc1-5e5b-4407-9123-89abd47119ce)

**Reply by g-bougard** [reply]
Indeed, empty "Agent base URL" here means the agent will reuse the one it is configured on. It would have been wrong if you had an URL set and which was not pointing on the plugin. It appeared people had a wrong url after a migration from FI plugin. So you can leave it empty.

**Comment by marxxtone** [comment]
Hi, LOG after Force inventory:

[Fri Sep 29 07:38:06 2023][debug] Logger backend File initialized
[Fri Sep 29 07:38:06 2023][debug] GLPI Agent (1.5)
[Fri Sep 29 07:38:06 2023][debug] Configuration directory: C:/Program Files/GLPI-Agent/etc
[Fri Sep 29 07:38:06 2023][debug] Data directory: C:/Program Files/GLPI-Agent/share
[Fri Sep 29 07:38:06 2023][debug] Storage directory: C:\Program Files\GLPI-Agent\var
[Fri Sep 29 07:38:06 2023][debug] Lib directory: C:/Program Files/GLPI-Agent/perl/agent
[Fri Sep 29 07:38:06 2023][debug] [target server0] Next server contact planned for Sat Sep 30 07:06:15 2023
[Fri Sep 29 07:38:06 2023][debug2] getAvailableTasks() : add of task Collect version 2.9
[Fri Sep 29 07:38:06 2023][debug2] getAvailableTasks() : add of task Deploy version 3.0
[Fri Sep 29 07:38:06 2023][debug2] getAvailableTasks() : add of task Inventory version 1.13
[Fri Sep 29 07:38:06 2023][debug2] getAvailableTasks() : add of task RemoteInventory version 1.2
[Fri Sep 29 07:38:06 2023][debug] Available tasks:
[Fri Sep 29 07:38:06 2023][debug] - Collect: 2.9
[Fri Sep 29 07:38:06 2023][debug] - Deploy: 3.0
[Fri Sep 29 07:38:06 2023][debug] - Inventory: 1.13
[Fri Sep 29 07:38:06 2023][debug] - RemoteInventory: 1.2
[Fri Sep 29 07:38:06 2023][debug] target server0: server https://MYDOMAIN/marketplace/glpiinventory/
[Fri Sep 29 07:38:06 2023][debug] Planned tasks for server0: RemoteInventory,Deploy,Collect,Inventory
[Fri Sep 29 07:38:06 2023][debug] Provided by Teclib Edition
[Fri Sep 29 07:38:06 2023][debug] Installer built on Wed Jun 21 14:27:07 2023 UTC
[Fri Sep 29 07:38:06 2023][debug] Built with Strawberry Perl 5.36.0
[Fri Sep 29 07:38:06 2023][debug] Built on github actions windows image for glpi-project/glpi-agent repository
[Fri Sep 29 07:38:06 2023][info] GLPI Agent service starting
[Fri Sep 29 07:38:06 2023][debug] Trusted target ip: MYIP
[Fri Sep 29 07:38:06 2023][debug] Trusted client ip/range: 127.0.0.1/32, ::ffff:7f00:1/128, ::1/128
[Fri Sep 29 07:38:06 2023][debug] [http server] Trying to load BasicAuthentication Server plugin
[Fri Sep 29 07:38:06 2023][debug] [basic authentication server plugin] Initializing BasicAuthentication v1.0 Server plugin...
[Fri Sep 29 07:38:06 2023][debug] [basic authentication server plugin] Loading BasicAuthentication Server plugin configuration from C:/Program Files/GLPI-Agent/etc/basic-authentication-server-plugin.cfg
[Fri Sep 29 07:38:06 2023][debug] [http server] HTTPD BasicAuthentication Server plugin loaded but disabled
[Fri Sep 29 07:38:06 2023][debug] [http server] Trying to load Inventory Server plugin
[Fri Sep 29 07:38:06 2023][debug] [inventory server plugin] Initializing Inventory v1.1 Server plugin...
[Fri Sep 29 07:38:06 2023][debug] [inventory server plugin] Loading Inventory Server plugin configuration from C:/Program Files/GLPI-Agent/etc/inventory-server-plugin.cfg
[Fri Sep 29 07:38:06 2023][debug] [http server] HTTPD Inventory Server plugin loaded but disabled
[Fri Sep 29 07:38:06 2023][debug] [http server] Trying to load Proxy Server plugin
[Fri Sep 29 07:38:07 2023][debug] [proxy server plugin] Initializing Proxy v2.2 Server plugin...
[Fri Sep 29 07:38:07 2023][debug] [proxy server plugin] Loading Proxy Server plugin configuration from C:/Program Files/GLPI-Agent/etc/proxy-server-plugin.cfg
[Fri Sep 29 07:38:07 2023][debug] [http server] HTTPD Proxy Server plugin loaded but disabled
[Fri Sep 29 07:38:07 2023][debug] [http server] Trying to load SecondaryProxy Server plugin
[Fri Sep 29 07:38:07 2023][debug] [proxy2 server plugin] Initializing SecondaryProxy v1.1 Server plugin...
[Fri Sep 29 07:38:07 2023][debug] [proxy2 server plugin] Loading SecondaryProxy Server plugin configuration from C:/Program Files/GLPI-Agent/etc/proxy2-server-plugin.cfg
[Fri Sep 29 07:38:07 2023][debug] [http server] HTTPD SecondaryProxy Server plugin loaded but disabled
[Fri Sep 29 07:38:07 2023][debug] [http server] Trying to load SSL Server plugin
[Fri Sep 29 07:38:07 2023][debug] [ssl server plugin] Initializing SSL v1.1 Server plugin...
[Fri Sep 29 07:38:07 2023][debug] [ssl server plugin] Loading SSL Server plugin configuration from C:/Program Files/GLPI-Agent/etc/ssl-server-plugin.cfg
[Fri Sep 29 07:38:07 2023][debug] [http server] HTTPD SSL Server plugin loaded but disabled
[Fri Sep 29 07:38:07 2023][debug] [http server] Trying to load Test Server plugin
[Fri Sep 29 07:38:07 2023][debug] [server test plugin] Initializing Test v1.5 Server plugin...
[Fri Sep 29 07:38:07 2023][debug] [server test plugin] Loading Test Server plugin configuration from C:/Program Files/GLPI-Agent/etc/server-test-plugin.cfg
[Fri Sep 29 07:38:07 2023][debug] [http server] HTTPD Test Server plugin loaded but disabled
[Fri Sep 29 07:38:07 2023][debug] [http server] Trying to load ToolBox Server plugin
[Fri Sep 29 07:38:07 2023][debug] [toolbox plugin] Initializing ToolBox v1.1 Server plugin...
[Fri Sep 29 07:38:07 2023][debug] [toolbox plugin] Loading ToolBox Server plugin configuration from C:/Program Files/GLPI-Agent/etc/toolbox-plugin.cfg
[Fri Sep 29 07:38:07 2023][debug] [http server] HTTPD ToolBox Server plugin loaded but disabled
[Fri Sep 29 07:38:07 2023][info] [http server] HTTPD service started on port 62354
[Fri Sep 29 07:38:07 2023][debug2] Looking for Win32_Processor class WMI objects
[Fri Sep 29 07:38:08 2023][debug] Agent memory usage before freeing memory: WSS=119140352 PFU=108593152
[Fri Sep 29 07:38:08 2023][info] GLPI Agent memory usage: WSS=3059712 PFU=108593152
[Fri Sep 29 07:38:08 2023][debug] Running in background mode
[Fri Sep 29 07:38:08 2023][info] target server0: next run: Sat Sep 30 07:06:15 2023 - https://MYDOMAIN/marketplace/glpiinventory/
[Fri Sep 29 07:38:18 2023][debug] target server0: init event for RemoteInventory task
[Fri Sep 29 07:38:19 2023][debug] target server0: init event for Deploy task
[Fri Sep 29 07:38:19 2023][debug] [target server0] New storage maintenance event on deploy task
[Fri Sep 29 07:38:19 2023][debug2] [target server0] Event scheduled in 120 seconds
[Fri Sep 29 07:38:20 2023][debug] target server0: init event for Collect task
[Fri Sep 29 07:38:21 2023][debug] target server0: init event for Inventory task

**Comment by marxxtone** [comment]
I think when I started Agent 1.5 without SSL in beginning, I saw in the Agent monitor after Force Inventory taskMaintenance also taskCollect and taskDeploy message. Now is taskMaintenance and then waiting in status message

**Comment by g-bougard** [comment]
Hi @marxxtone 
you think you forced inventory, but you didn't:
```
...
[Fri Sep 29 07:38:06 2023][debug] [target server0] Next server contact planned for Sat Sep 30 07:06:15 2023
...
...
[Fri Sep 29 07:38:08 2023][info] target server0: next run: Sat Sep 30 07:06:15 2023 - https://mydomain/marketplace/glpiinventory/
...
```
On these lines, the agent is telling you it will run the inventory and all other tasks only in one day.

From the computer itself, you can access `http://localhost:62354` and click on `Run now`. I see in your log, I see localhost is effectively trusted.
You can also try to force inventory from the asset page itself in GLPI in the case the agent 62354 port is not filtered from the GLPI point of view:
![image](https://github.com/glpi-project/glpi-agent/assets/12514489/38d108a6-bc41-4fc7-ba05-e65b2b624a2d)

**Comment by marxxtone** [comment]
Hi, There is no problem with the inventory. The request from the GLPI server and to localhost:62345 works fine.

![image](https://github.com/glpi-project/glpi-agent/assets/145987827/e90d629a-bfcb-4feb-968d-ea38725bf365)

**Reply by g-bougard** [reply]
And so ? What do you see in the agent log after you forced tasks run this way ? Remember to force the Deploy task in the plugin before forcing the agent.

**Comment by marxxtone** [comment]
Here is log file from host after inventory:

```
[Fri Sep 29 13:47:03 2023][debug] [http server] GET request / from client 127.0.0.1
[Fri Sep 29 13:47:03 2023][debug] Trusted target ip: 192.168.2.130/32, ::ffff:c0a8:282/128
[Fri Sep 29 13:47:03 2023][debug] Trusted client ip/range: 127.0.0.1/32, ::ffff:7f00:1/128, ::1/128
[Fri Sep 29 13:47:03 2023][debug] [http server] response status 200
[Fri Sep 29 13:47:25 2023][debug] [http server] GET request /now from client 127.0.0.1
[Fri Sep 29 13:47:25 2023][debug] [http server] rescheduling next contact for all targets right now
[Fri Sep 29 13:47:25 2023][debug] [http server] response status 200
[Fri Sep 29 13:47:26 2023][info] target server0: server https://MYDOMAIN/marketplace/glpiinventory/
[Fri Sep 29 13:47:26 2023][debug2] [http client] Using Compress::Zlib for compression
[Fri Sep 29 13:47:26 2023][info] sending contact request to server0
[Fri Sep 29 13:47:26 2023][debug2] [http client] 62F74BCF: sending message:
{
   "action": "contact",
   "deviceid": "XXX.mydomain-2023-09-29-09-31-52",
   "enabled-tasks": [
      "collect",
      "deploy",
      "inventory",
      "remoteinventory"
   ],
   "httpd-plugins": {},
   "httpd-port": "62354",
   "installed-tasks": [
      "collect",
      "deploy",
      "inventory",
      "remoteinventory"
   ],
   "name": "GLPI-Agent",
   "version": "1.5"
}
[Fri Sep 29 13:47:26 2023][debug2] format: Zlib
[Fri Sep 29 13:47:26 2023][debug2] [http client] 62F74BCF: received message:
{"message":"remoteinventory task not supported","disabled":["remoteinventory"],"expiration":"24","status":"ok","tasks":{"collect":
{"version":"1.2.3","server":"glpiinventory"},"deploy":{"version":"1.2.3","server":"glpiinventory"},"inventory":{"server":"glpi","version":"10.0.9"}}}
[Fri Sep 29 13:47:26 2023][debug] server message: remoteinventory task not supported
[Fri Sep 29 13:47:26 2023][debug2] [http client] Using Compress::Zlib for compression
[Fri Sep 29 13:47:26 2023][info] sending prolog request to server0
[Fri Sep 29 13:47:26 2023][debug2] [http client] sending message:
<?xml version="1.0" encoding="UTF-8"?>
<REQUEST>
  <DEVICEID>xxx.mydomain-2023-09-29-09-31-52</DEVICEID>
  <QUERY>PROLOG</QUERY>
  <TOKEN>12345678</TOKEN>
</REQUEST>
[Fri Sep 29 13:47:26 2023][debug2] format: Zlib
[Fri Sep 29 13:47:26 2023][debug2] [http client] receiving message:
<?xml version="1.0"?>
<REPLY><PROLOG_FREQ>24</PROLOG_FREQ><RESPONSE>SEND</RESPONSE></REPLY>
[Fri Sep 29 13:47:26 2023][debug] new thread 8 to handle task Deploy
[Fri Sep 29 13:47:26 2023][info] running task Deploy
[Fri Sep 29 13:47:26 2023][debug2] [http client] Not using compression
[Fri Sep 29 13:47:26 2023][debug2] https://mydomain/marketplace/glpiinventory/?action=getConfig&machineid=xxx.mydomain-2023-09-29-09-31-52&task[Deploy]=3.0
[Fri Sep 29 13:47:26 2023][debug] [http server] GET request / from client 127.0.0.1
[Fri Sep 29 13:47:26 2023][debug] [http server] response status 200
[Fri Sep 29 13:47:26 2023][debug2] Looking for Win32_LogicalDisk class WMI objects
[Fri Sep 29 13:47:26 2023][debug] Free space on C:\Program Files\GLPI-Agent\var\https..__mydomain_marketplace_glpiinventory_\deploy: 204619
[Fri Sep 29 13:47:26 2023][debug2] https://mydoman/marketplace/glpiinventory/b/deploy/?action=getJobs&machineid=xxx.mydomain023-09-29-09-31-52&version=3.0
[Fri Sep 29 13:47:26 2023][debug2] Deploy job 6516b8bfe3d3c in the list
[Fri Sep 29 13:47:26 2023][debug2] Processing job 6516b8bfe3d3c from the list
[Fri Sep 29 13:47:26 2023][debug2] https://mydomain/marketplace/glpiinventory/b/deploy/?action=setStatus&msg=starting&machineid=xxx.mydomain-2023-09-29-09-31-52&uuid=6516b8bfe3d3c&part=job&currentStep=checking
[Fri Sep 29 13:47:26 2023][debug2] Checking job 6516b8bfe3d3c...
[Fri Sep 29 13:47:26 2023][debug2] https://mydomain/marketplace/glpiinventory/b/deploy/?action=setStatus&machineid=xxx.mydomain-2023-09-29-09-31-52&uuid=6516b8bfe3d3c&part=job&currentStep=checking&msg=all%20checks%20are%20ok&status=ok
[Fri Sep 29 13:47:26 2023][debug2] usercheck before: no user interaction requested
[Fri Sep 29 13:47:26 2023][debug2] Downloading for job 6516b8bfe3d3c...
[Fri Sep 29 13:47:26 2023][debug2] https://mydomain/marketplace/glpiinventory/b/deploy/?action=setStatus&msg=downloading%20files&machineid=xxx.mydomain-2023-09-29-09-31-52&uuid=6516b8bfe3d3c&part=job&currentStep=downloading
[Fri Sep 29 13:47:26 2023][debug2] https://mydomain/marketplace/glpiinventory/b/deploy/?action=setStatus&sha512=f45ce2be338215a46125a145eb9ab2ec384aad4881c39b0169ef1dc0e70649c596c13729f9cd08150cabffa1ebc1a1d3b7ca51c389d1440c5ae2e6c555734360&machineid=xxx.mydomain-2023-09-29-09-31-52&uuid=6516b8bfe3d3c&part=file&currentStep=downloading&msg=fetching%20googlechromestandaloneenterprise64.msi
[Fri Sep 29 13:47:26 2023][info] looking for a peer in the network
[Fri Sep 29 13:47:26 2023][debug2] Looking for Win32_NetworkAdapterConfiguration class WMI objects
[Fri Sep 29 13:47:26 2023][debug2] Looking for MSFT_NetAdapter class WMI objects
[Fri Sep 29 13:47:27 2023][debug] Scanning from 192.168.2.0 to 192.168.2.255
[Fri Sep 29 13:47:27 2023][debug] Scanning from 192.168.2.57 to 192.168.2.28
[Fri Sep 29 13:47:29 2023][debug] [http server] GET request /status from client 127.0.0.1
[Fri Sep 29 13:47:29 2023][debug] [http server] response status 200
[Fri Sep 29 13:47:29 2023][debug] [http server] GET request /status from client 127.0.0.1
[Fri Sep 29 13:47:29 2023][debug] [http server] response status 200
[Fri Sep 29 13:47:31 2023][debug] [http server] GET request /status from client 127.0.0.1
[Fri Sep 29 13:47:31 2023][debug] [http server] response status 200
[Fri Sep 29 13:47:33 2023][debug] [http server] GET request /status from client 127.0.0.1
[Fri Sep 29 13:47:33 2023][debug] [http server] response status 200
[Fri Sep 29 13:47:35 2023][debug] [http server] GET request /status from client 127.0.0.1
[Fri Sep 29 13:47:35 2023][debug] [http server] response status 200
[Fri Sep 29 13:47:36 2023][debug] File part URL: http://192.168.2.57:62354/deploy/getFile/a/a4/a4920b24cbca64c35319bef1626afcf4f242f93d8acdb5f6a2e9fee39854b01f090d52f5da35a6e8e99a60062fa7d003ed0bdbd6ad95f4c8ff375a8000262249
[Fri Sep 29 13:47:36 2023][error] [http client] communication error: 404 Not Found
[Fri Sep 29 13:47:36 2023][debug] File part URL: http://192.168.2.23:62354/deploy/getFile/a/a4/a4920b24cbca64c35319bef1626afcf4f242f93d8acdb5f6a2e9fee39854b01f090d52f5da35a6e8e99a60062fa7d003ed0bdbd6ad95f4c8ff375a8000262249
```

**Comment by marxxtone** [comment]
Status: downloading files:

![image](https://github.com/glpi-project/glpi-agent/assets/145987827/0680baba-0f43-46a2-bc55-967fb451ef2d)




Agent still trying to communicate:

 [Fri Sep 29 14:03:15 2023][error] [http client] communication error: 404 Not Found

[Fri Sep 29 14:03:15 2023][debug] File part URL: http://192.168.2.12:62354/deploy/getFile/9/9e/9ef7c6593ac596604c82b89af038feb774754be6714490de0256f7f75cc9be209e1420edd559c18e01b38dfdf0d357b352cc3c461eedbea21612b747d9eed727
![image](https://github.com/glpi-project/glpi-agent/assets/145987827/835d1c09-2532-4d33-b0d8-bd9549137508)

[Fri Sep 29 14:03:13 2023][error] [http client] communication error: 404 Not Found
[Fri Sep 29 14:03:13 2023][debug] File part URL: http://192.168.2.46:62354/deploy/getFile/9/9e/9ef7c6593ac596604c82b89af038feb774754be6714490de0256f7f75cc9be209e1420edd559c18e01b38dfdf0d357b352cc3c461eedbea21612b747d9eed727
[Fri Sep 29 14:03:13 2023][error] [http client] communication error: 404 Not Found
[Fri Sep 29 14:03:13 2023][debug] File part URL: http://192.168.2.85:62354/deploy/getFile/9/9e/9ef7c6593ac596604c82b89af038feb774754be6714490de0256f7f75cc9be209e1420edd559c18e01b38dfdf0d357b352cc3c461eedbea21612b747d9eed727
[Fri Sep 29 14:03:13 2023][error] [http client] communication error: 404 Not Found
[Fri Sep 29 14:03:13 2023][debug] File part URL: http://192.168.2.13:62354/deploy/getFile/9/9e/9ef7c6593ac596604c82b89af038feb774754be6714490de0256f7f75cc9be209e1420edd559c18e01b38dfdf0d357b352cc3c461eedbea21612b747d9eed727
[Fri Sep 29 14:03:13 2023][error] [http client] communication error: 404 Not Found
[Fri Sep 29 14:03:13 2023][debug] File part URL: http://192.168.2.99:62354/deploy/getFile/9/9e/9ef7c6593ac596604c82b89af038feb774754be6714490de0256f7f75cc9be209e1420edd559c18e01b38dfdf0d357b352cc3c461eedbea21612b747d9eed727
[Fri Sep 29 14:03:13 2023][error] [http client] communication error: 404 Not Found
[Fri Sep 29 14:03:13 2023][debug] File part URL: http://192.168.2.106:62354/deploy/getFile/9/9e/9ef7c6593ac596604c82b89af038feb774754be6714490de0256f7f75cc9be209e1420edd559c18e01b38dfdf0d357b352cc3c461eedbea21612b747d9eed727
[Fri Sep 29 14:03:13 2023][error] [http client] communication error: 404 Not Found
[Fri Sep 29 14:03:13 2023][debug] File part URL: http://192.168.2.26:62354/deploy/getFile/9/9e/9ef7c6593ac596604c82b89af038feb774754be6714490de0256f7f75cc9be209e1420edd559c18e01b38dfdf0d357b352cc3c461eedbea21612b747d9eed727
[Fri Sep 29 14:03:13 2023][error] [http client] communication error: 404 Not Found

**Comment by marxxtone** [comment]
Could there be a problem with the .msi package and the ACTION command to run it?

**Comment by g-bougard** [comment]
Actually, as you enabled p2p, the agent is first asking all peer it found in its ip-range for the required file part. If this is the first agent which requests this part, all peer will answer 404 not found and the agent will finally ask the part to the server. It will than host that file part for other peers.
Be aware p2p is really an advantage when the link between peers and GLPI is really slow.

So the task is running and may take some time, wait until it terminates and report what's the result and how many time it took.

**Comment by marxxtone** [comment]
The agent is constantly trying to call IP stations on the LAN. After about 30 minutes I turned it off. It does it in a loop. Apparently it doesn't reach the server at all. 

I tried disabling P2P on the Agent and rebooting the station. It's the same.

Nice, thank you for your direction to win! 
I will try it. Nice weekend!

**Comment by marxxtone** [comment]
I had to turn on SERVER MIRROR and reinstall the agent on a new station. Everything is fine! Thank you for the additional information to solve the problem!

![image](https://github.com/glpi-project/glpi-agent/assets/145987827/a4d94d96-2c53-4a0a-9480-63b86d497561)
